build.gradle

```diff
@@ -323,6 +324,8 @@ dependencies {
     implementation group: 'org.springframework', name: 'spring-tx', version: versions.springVersion
     implementation group: 'org.springframework', name: 'spring-web', version: versions.springVersion
     implementation group: 'org.springframework', name: 'spring-webmvc', version: versions.springVersion
+    
+    implementation group: 'org.bouncycastle', name: 'bcpkix-jdk15on', version: '1.70'
```

Comment:
why jdk15

Reply:
I used the same dependency that @manukundloo-hmcts used in his PR: https://github.com/hmcts/rd-judicial-api/pull/458